hieradata/rspec/role_jenkins.yaml

```diff
@@ -0,0 +1,186 @@
+---
+lookup_options:
+  "^profile::jenkinscontroller::jcasc$":
+    merge:
+      strategy: deep
+      merge_hash_arrays: true
+
+profile::jenkinscontroller::ci_fqdn: 'jenkins.role.respec.test.local'
+# profile::jenkinscontroller::proxy_port: 443
```

Comment:
Why all these commented lines?

Reply:
it was me-PEBKAC .

Fixed :) Thanks for tha catch